CHANGELOG.rst

```diff
@@ -6,6 +6,275 @@
 Synapse Changelog
 *****************
 
+v2.172.0 - 2024-06-22
+=====================
+
+Model Changes
+-------------
+- Updates to the ``biz``, ``econ``, ``inet``, ``meta``, ``ou`` ``risk``,
+ and ``transit`` models.
+  (`#3561 <https://github.com/vertexproject/synapse/pull/3561>`_)
+  (`#3756 <https://github.com/vertexproject/synapse/pull/3756>`_)
+
+  **New Interfaces**
+
+  ``inet:service:base``
+    Properties common to most forms within a service platform.
+
+  ``inet:service:object``
+    Properties common to objects within a service platform. This inherits
+    from the ``inet:service:base`` interface.
+
+  **New Forms**
```

Comment:
Feel free to say no:

Can we sort this section lexicographically by form name? It would help for visually scanning for a specific form. Probably the "New Properties" section below too.